### ISSUE TITLE
Slow and low quality inference

### ISSUE THREAD

**Issue by rhystz** [ShmuelRonen/ComfyUI-ThinkSound_Wrapper#3]
Hi. I'm getting much slower than expected inference times, and seemingly low quality outputs (poor sound, out of sync). Is there an optimum FPS and video resolution? Is the added 2 seconds (using math node) in your workflow important?

Thanks for any help.

---

[ComfyUI-Manager] All startup tasks have been completed.
🔍 DEBUG: Starting ThinkSound import process...
🔍 DEBUG: Script directory = C:\etc - Copy\ComfyUI\custom_nodes\ComfyUI-ThinkSound_Wrapper
🔍 DEBUG: Added to sys.path: C:\etc - Copy\ComfyUI\custom_nodes\ComfyUI-ThinkSound_Wrapper
🔍 DEBUG: Contents of script_directory:
  📁 .git/
  📄 .gitignore
  📁 ckpts/
  📁 configs/
  📁 data_utils/
  📁 example_workflows/
  📄 nodes.py
  📄 README.md
  📄 requirements.txt
  📁 thinksound/
  📄 __init__.py
  📁 __pycache__/
🔍 DEBUG: Trying ThinkSound module import...
🔍 DEBUG: thinksound subfolder = C:\etc - Copy\ComfyUI\custom_nodes\ComfyUI-ThinkSound_Wrapper\thinksound
🔍 DEBUG: thinksound subfolder exists = True
🔍 DEBUG: Contents of thinksound subfolder:
  📁 configs
  📁 data
  📁 inference
  📁 models
  📁 training
  📁 __init__.py
  📁 __pycache__
🔍 DEBUG: Created ThinkSound alias for thinksound module
✅ SUCCESS: Found FeaturesUtils in thinksound.data.v2a_utils.feature_utils_224
🔍 DEBUG: Trying models import...
✅ SUCCESS: Found create_model_from_config in thinksound.models.factory
🔍 DEBUG: Trying utils import...
✅ SUCCESS: Found load_ckpt_state_dict in thinksound.models.utils
🔍 DEBUG: Trying sampling import...
✅ SUCCESS: Found sampling functions in thinksound.inference.sampling
🎉 ThinkSound modules imported successfully!

---

got prompt
🔍 DEBUG: Using ComfyUI models directory: C:etc - Copy\ComfyUI\models
❌ Local MetaCLIP not found at C:etc - Copy\ComfyUI\models\thinksound\metaclip-h14-fullcc2.5b, using online
❌ Local T5 not found at C:etc - Copy\ComfyUI\models\t5-v1_1-xl, using online
🔄 Loading MetaCLIP model from: facebook/metaclip-h14-fullcc2.5b
🔄 Loading MetaCLIP processor from: facebook/metaclip-h14-fullcc2.5b
Using a slow image processor as `use_fast` is unset and a slow processor was saved with this model. `use_fast=True` will be the default behavior in v4.52, even if the model was saved with a slow processor. This will result in minor differences in outputs. You'll still be able to use a slow processor with `use_fast=False`.
✅ MetaCLIP model and processor loaded successfully
🔄 Loading T5 tokenizer from: google/t5-v1_1-xl
You are using the default legacy behaviour of the <class 'transformers.models.t5.tokenization_t5.T5Tokenizer'>. This is expected, and simply means that the `legacy` (previous) behavior will be used so nothing changes for you. If you want to use the new behaviour, set `legacy=False`. This should only be set if you understand what it means, and thoroughly read the reason why this was added as explained in https://github.com/huggingface/transformers/pull/24565
🔄 Loading T5 model from: google/t5-v1_1-xl
✅ T5 model and tokenizer loaded successfully
🔄 Loading Synchformer from: C:etc - Copy\ComfyUI\models\thinksound\synchformer_state_dict.pth
Loading MotionFormer config from C:etc - Copy\ComfyUI\custom_nodes\ComfyUI-ThinkSound_Wrapper\data_utils\ext\synchformer\divided_224_16x4.yaml
✅ Synchformer loaded successfully
✅ Loaded ThinkSound FeatureUtils on cuda:0 with dtype torch.float32
flash_attn installed, using Flash Attention
C:etc - Copy\python_embeded\Lib\site-packages\vector_quantize_pytorch\vector_quantize_pytorch.py:436: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled = False)
C:etc - Copy\python_embeded\Lib\site-packages\vector_quantize_pytorch\vector_quantize_pytorch.py:619: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled = False)
C:etc - Copy\python_embeded\Lib\site-packages\torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
✅ Model loaded successfully
Loaded ThinkSound model weights from C:etc - Copy\ComfyUI\models\thinksound\thinksound_light.ckpt
Video too short: 200 frames for 10.0s
Processed video: clip_frames torch.Size([1, 80, 3, 224, 224]), sync_frames torch.Size([1, 200, 3, 224, 224]), duration 8.0
Models moved to cuda:0

**Comment by rhystz** [comment]
https://github.com/user-attachments/assets/50a2aea1-14cb-4a8b-a90c-a30b2338dc7d

Is this an expected level of quality with a low quality prompt?
